autoload/codeclimate.vim

```diff
@@ -92,25 +92,30 @@ function! s:RunAnalysis(files)
     echohl None
     return
   endif
+  let l:issues = s:ParseIssues(l:analyze_output)
+  if 0 == len(l:issues)
+    echo 'Code Climate found no issues! Well done.'
+  else
+    call s:ShowIssues(l:issues)
+  endif
+endfunction
+
+function! s:ParseIssues(text) abort
```

Comment:
If/when Neovim has native Lua support, I'd consider deprecating the vimscript version of this plugin & rewriting it in Lua. At that point, I'd be more likely to switch to JSON parsing.